### DISCUSSION TITLE
service: floorplan.*

### DISCUSSION THREAD

**Discussion by DeFiEcoSys** [ExperienceLovelace/ha-floorplan#243] [Q&A]
Hi,

Hope you're all well. I've installed the front end through HACS, and then added in the resources.

The service call methods aren't in my developer section (and hence I think why my config doesn't work)

**Comment by OzGav** [comment]
I am not sure what you mean by your last sentence. Just to be clear though HA is only able to be configured via YAML not the UI.

**Comment by DeFiEcoSys** [comment]
Thanks @OzGav , I'm really struggling with getting things setup and working.

What my last sentence means is what I think the problem is. Basically the "call-service" function allows any of the HA services to be called, this list can be found in the Developer Tools -> Service. i.e. you can select "Switch: Toggle" and then select the entity and click "Call Service" and the switch will change from On -> Off (or vica versa).

In the case here on FloorPlan, the YAML wants to perform call-service with this method "service: floorplan.style_set" (i.e. Floorplan.*) in order to do the changes (see below extract from home example)

rules:
      - entity: light.garage
        element: light.garage
        state_action:
          action: call-service
          service: floorplan.image_set
          service_data:
            image: /local/floorplan/examples/home/light_${entity.state}.svg
            cache: true
        tap_action: toggle

^^ the above (from home example in docs) will call the service "floorplan.image_set", *BUT* if I go and look in my Developer Tools -> Services there isn't a service called Floorplan.[anything]. Is this mean to be here?

**Reply by OzGav** [reply]
No as these are floorplan specific service calls that only work within ha-floorplan

**Reply by DeFiEcoSys** [reply]
Thanks for clarifying. 

Back to the drawing board =/

**Comment by exetico** [comment]
Did you get things under control, or do you still need help? Let us know what you're currently fighting with :smiley: ...
